outbox/README.md

```diff
@@ -16,59 +16,83 @@ This allows to build a native binary of that service, resulting in significantly
 
 Prepare the Java components:
 
-    mvn clean install -Pnative -Dnative-image.docker-build=true
+```console
+$ mvn clean install -Pnative -Dnative-image.docker-build=true
+```
 
 Start all components:
 
-    export DEBEZIUM_VERSION=0.9
-    docker-compose up --build
+```console
+$ export DEBEZIUM_VERSION=0.9
+$ docker-compose up --build
+```
 
 Register the Debezium Postgres connector:
 
-    cat register-postgres.json  | http POST http://localhost:8083/connectors/
+```console
+$ http PUT http://localhost:8083/connectors/outbox-connector/config < register-postgres.json
```

Comment:
PUT is an idempotent operation, you can decide the name of the connector in the url and always post a new configuration, independently if it was created before or not. I tend to use this one because the `json` payload is uniform, if you want to do updates in the connector you only need a single payload now, instead of two

Reply:
Gotcha. Didn't know one could use PUT for creation, too. Thanks for sharing, I'll adopt that :)